admin-cli/src/index.ts

```diff
@@ -70,32 +70,31 @@ namespace v3 {
         console.log(`Accepted EmployeeProposal for ${employee}.`);
       }
     }
-
-    // Create/approve employee vacation requests.
+    // Create/approve vacation requests.
     for (const employee in config.employees) {
       const employeeInfo = config.employees[employee];
       for (const vacationRequest of employeeInfo.vacationRequests ?? []) {
+        const fromDate = vacationRequest.fromDate;
+        const toDate = vacationRequest.toDate;
```

Comment:
```suggestion
        const {toDate, fromDate} = vacationRequest;
```
and remove the line above.